public/scripts/services/form.js

```diff
@@ -25,6 +25,26 @@
                         value = null;
                     }
                     break;
+                case 'string-datetime':
+                    value = value.toString();
```

Comment:
What are the possible values that `value` can be?

Reply:
Good catch, always string as it's value form input. Removing toString()